requirements.txt

```diff
@@ -5,4 +5,5 @@ duckdb
 pyarrow
 pandas
 polars
-cdxj-indexer
\ No newline at end of file
+cdxj-indexer
+setuptools
```

Comment:
I found that on a pristine `Python 3.12.4` the following failed:

```
make cdx_toolkit
look up this capture in the comoncrawl cdx index
cdxt --cc --from 20240518015810 --to 20240518015810 iter an.wikipedia.org/wiki/Escopete
Traceback (most recent call last):
  File "/Users/jason/venv/whirlwind/bin/cdxt", line 5, in <module>
    from cdx_toolkit.cli import main
  File "/Users/jason/venv/whirlwind/lib/python3.12/site-packages/cdx_toolkit/__init__.py", line 3, in <module>
    from pkg_resources import get_distribution, DistributionNotFound
ModuleNotFoundError: No module named 'pkg_resources'
make: *** [cdx_toolkit] Error 1
```

Reply:
I had that in https://github.com/commoncrawl/whirlwind-python/pull/4, I'll close that in favor of this.